### PR TITLE
[10890] Verify implementation of wait_for_sidecar() checks for respon…

### PR DESCRIPTION
**Pull request by dennismeister93**
…se status code

Signed-off-by: Dennis Meister <dennis.meister@bosch-softtec.com>

## Describe your changes

Verify implementation of wait_for_sidecar() checks for response status code

## Issue ticket number and link

[AB#_[10890]_](https://softwaredefinedvehicle.visualstudio.com/SoftwareDefinedVehicle/_workitems/edit/10890)

## Checklist - Manual tasks

* [x] Examples are executing successfully
* [x] Devcontainer can be opened successfully
* [x] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully

### CODE REVIEW

**Review by emsameen — CHANGES_REQUESTED** [review]
I approve the changes, LGTM. However, I would move the status to pending until the contributor approval is ready and the source of the PR is adjusted.

**Comment by dennismeister93** [comment]
Closed due to official PR from SDV org https://github.com/eclipse-velocitas/vehicle-app-python-sdk/pull/11

**Pull request closed without merging**
